src/main/java/com/mraof/minestuck/effects/SuspicionEffect.java

```diff
@@ -0,0 +1,81 @@
+package com.mraof.minestuck.effects;
+
+import com.mraof.minestuck.Minestuck;
+import net.minecraft.world.effect.MobEffect;
+import net.minecraft.world.effect.MobEffectCategory;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraftforge.event.entity.EntityMountEvent;
+import net.minecraftforge.event.entity.living.ZombieEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import org.jetbrains.annotations.NotNull;
+
+@Mod.EventBusSubscriber(modid = Minestuck.MOD_ID, bus = Mod.EventBusSubscriber.Bus.FORGE)
+public class SuspicionEffect extends MobEffect
```

Comment:
Its a pretty simple class, but please add a short javadoc style description. We are trying to get into the habit of commenting our codebase to make things as quick to understand as possible